src/Schema/Directives/CanDirective.php

```diff
@@ -66,6 +71,18 @@ public static function definition(): string
   """
   injectArgs: Boolean = false
 
+  """
+  Resolve the model by directive args like @eq, @where etc.
+  You may not use the find argument in combination with findByArgs
+  """
+  findByArgs: Boolean = false
+
+  """
+  Apply scopes to the underlying query.
+  The scopes can only be used in combination with the findByArgs argument
```

Comment:
Nope, given you already added it in `modelsToCheck()`. It makes sense to enable it for both use cases.